### ISSUE TITLE
Snapware Airtight Leakproof Pyrex

### ISSUE THREAD

**Issue by nelsonic** [dwyl/health#31]
@iteles, as discussed, These: http://www.amazon.com/Snapware-Solution-Keeper-Featuring-18-Piece/dp/B00BSZJYY4
![amazon com-snapware-pyrex](https://cloud.githubusercontent.com/assets/194400/12122522/807b4cda-b3d3-11e5-9d10-2e60dd4c0194.png)

But **_not**_ at the UK price... http://www.amazon.co.uk/Snapware-Airtight-Leakproof-Pyrex-Keeper/dp/B00VRVYF4S/
![amazon-uk-snapware-airtight-pyrex](https://cloud.githubusercontent.com/assets/194400/12122559/c56f4170-b3d3-11e5-980d-c14cc4a7797f.png)

**Comment by iteles** [comment]
We _could_ get it from eBay, but this set mostly contains small containers, not sure how useful that will be: http://www.ebay.co.uk/itm/Snapware-Total-Solution-19-Piece-Pyrex-Glass-Food-Storage-Set-BRAND-NEW-/191737402286?hash=item2ca47093ae:g:FocAAOSwvt1WRVoh

Need to investigate a little!

**Comment by nelsonic** [comment]
Small containers are _useful_ in helping us reduce portion size...
I'm _almost_ tempted to get a new [small freezer](http://www.johnlewis.com/zanussi-zfg06400wa-freezer-a-energy-rating-50cm-wide-white/p475386?imageCount=2) and keep it in G8 just for prepared food... :open_mouth: 
Its a shame someone _nicked_ our old one... :disappointed:

**Comment by iteles** [comment]
Just spent a very pleasant 10 minute break looking through these :stuck_out_tongue: 
There are some 'Glasslock' options in [various sizes](http://www.amazon.co.uk/Glasslock-Pieces-Rectangular-Storage-GL-135/dp/B006Z58F6K/ref=pd_sim_201_4?ie=UTF8&dpID=41KzgREcbvL&dpSrc=sims&preST=_AC_UL160_SR160%2C160_&refRID=10M1CH7P0R17XRY3Z2XS) or [smallish ones](http://www.amazon.co.uk/Glasslock-Storage-rectangular-400ml-MCRB-040/dp/B00AQJTSOE/ref=sr_1_2?ie=UTF8&qid=1452093630&sr=8-2&keywords=glasslock) at 3 for £21-26,  
but the best option still looks to be the [8 pyrexes of various sizes](http://www.ebay.co.uk/itm/Snapware-Total-Solution-19-Piece-Pyrex-Glass-Food-Storage-Set-BRAND-NEW-/191737402286?hash=item2ca47093ae:g:FocAAOSwvt1WRVoh) for £50

I'm up for buying them!

**Comment by nelsonic** [comment]
Yeah, the eBay ones look good. lets order them when we get the contract. :wink:

**Comment by iteles** [comment]
:+1: Snapware and a trip to Ikea, this contract is looking better and better! :see_no_evil:

**Comment by nelsonic** [comment]
And when we get _paid_ we get to book a trip...
granted its "only" to visit family, but I'm excited! :wink:
